core/controllers/reader.py

```diff
@@ -571,7 +571,7 @@ def post(self, exploration_id):
             collection_services.record_played_exploration_in_collection_context(
                 user_id, collection_id, exploration_id)
             collections_left_to_complete = (
```

Comment:
This varname should be changed, perhaps?

Reply:
Done